packages/x-date-pickers/src/locales/huHU.ts

```diff
@@ -0,0 +1,67 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+
+// maps TimeView to its translation
+const timeViews = {
+  hours: 'Óra',
+  minutes: 'Perc',
+  seconds: 'Másodperc',
+};
+
+const huHUPickers: Partial<PickersLocaleText<any>> = {
+  // Calendar navigation
+  previousMonth: 'Előző hónap',
+  nextMonth: 'Következő hónap',
+
+  // View navigation
+  openPreviousView: 'Előző nézet megnyitása',
+  openNextView: 'Következő nézet megnyitása',
+  calendarViewSwitchingButtonAriaLabel: (view) =>
+    view === 'year'
+      ? 'az évválasztó már nyitva, váltson a naptárnézetre'
+      : 'a naptárnézet már nyitva, váltson az évválasztóra',
+  // inputModeToggleButtonAriaLabel: (isKeyboardInputOpen: boolean, viewType: 'calendar' | 'clock') => isKeyboardInputOpen ? `text input view is open, go to ${viewType} view` : `${viewType} view is open, go to text input view`,
```

Comment:
@noherczeg Would you be willing to translate this label?
It has been removed in v6, that's why when back-porting this change to v5 (master) we either leave with one missing label or the need to provide it's translation.

Reply:
@LukasTy sure, on it.

Reply:
Yeah, we can easily add it in ourselves. 👌 
Just a note, that In this case `viewType` is once again a variable name.
For a complete translation you can refer to other locales, like for example [csCZ](https://github.com/mui/mui-x/blob/master/packages/x-date-pickers/src/locales/csCZ.ts#L29). 😉